src/modelutils.jl

```diff
@@ -1026,13 +1037,25 @@ function one2prev_generator(topic_words::Matrix{Int}, vocab::Dict{Int, String},
     Channel{Tuple{Int, Int}}(buffer) do ch
 		for i in CartesianIndices(topic_words)
 			row, col = i[1], i[2]
+			current_tok = topic_words[i]
+
+			## Skip this topic word if it is not found in the vocabulary
```

Comment:
This check and the one on line 1053 make sure to skip if the topic words are missing from vocabulary. If everything is missing from vocabulary, then no pairs are generated and line 1026 throws an error.

Reply:
Sure, no rush, you can revisit when you're able to.